sklearn/metrics/classification.py

```diff
@@ -1875,7 +1875,8 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
     y_true : array-like or label indicator matrix
         Ground truth (correct) labels for n_samples samples.
 
-    y_pred : array-like of float, shape = (n_samples, n_classes) or (n_samples,)
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Reply:
@jnothman I did that because otherwise it did not pass the flake8 test.

Reply:
We don't require that you fix pep8 problems in existing code

Reply:
ok, good to know.  I was wondering how it passed the previous tests.

Reply:
Usually we would ask that you undo these changes, but it's not a big deal here IMO

Reply:
Do 2 reviewers have to approve the changes before the PR is merged in?

Reply:
Ordinarily, yes